allure-generator/src/main/javascript/components/widget-status/WidgetStatusView.hbs

```diff
@@ -15,7 +15,7 @@
         {{#if (gt total items.length)}}
             <a class="table__row" href="#{{baseUrl}}">
                 <div class="table__col table__col_center">
-                    {{showAllText}}
+                    {{t showAllText}}
```

Comment:
`{{t 'component.widgetStatus.showAll'}}`

and add translations to https://github.com/allure-framework/allure2/tree/master/allure-generator/src/main/javascript/translations